submarine-workbench/submarine-server/src/test/java/org/apache/submarine/database/service/TeamServiceTest.java

```diff
@@ -0,0 +1,145 @@
+    teamMember.setTeamName("submarine");
+    teamMember.setInviter(0);
+    teamMember.setMember("admin");
+    teamMember.setCreateBy("createByteamMember");
+
+    Team team = new Team();
+    team.setTeamName("submarine");
+    team.setOwner("zhulinhao");
+    team.setCreateBy("createByteam");
+    List list = new ArrayList<TeamMember>();
+    list.add(teamMember);
+    team.setCollaborators(list);
+    Boolean ret = teamService.add(team);
+    assertTrue(ret);
+
+    List<Team> teamList = teamService.queryPageList("zhulinhao", "create_time", "desc", 0, 100);
+    assertEquals(teamList.size(), 1);
+    Team team_db = teamList.get(0);
+    assertEquals(team.getTeamName(), team_db.getTeamName());
+    assertEquals(team.getOwner(), team_db.getOwner());
+    assertEquals(team.getCreateBy(), team_db.getCreateBy());
+
+    assertEquals(team_db.getCollaborators().size(), 1);
+    TeamMember teamMember_db = team_db.getCollaborators().get(0);
+    assertEquals(team.getId(), teamMember_db.getTeamId());
+    assertEquals(teamMember.getTeamName(), teamMember_db.getTeamName());
+    assertEquals(teamMember.getInviter(), teamMember_db.getInviter());
+    assertEquals(teamMember.getMember(), teamMember_db.getMember());
+    assertEquals(teamMember.getCreateBy(), teamMember_db.getCreateBy());
+  }
+
+  @Test
+  public void updateByPrimaryKeySelective() throws Exception {
+    TeamMember teamMember = new TeamMember();
+    teamMember.setTeamName("submarine");
+    teamMember.setInviter(0);
+    teamMember.setMember("admin");
+    teamMember.setCreateBy("createByteamMember");
+
+    Team team = new Team();
+    team.setTeamName("submarine");
+    team.setOwner("zhulinhao");
+    team.setCreateBy("createByteam");
+    List list = new ArrayList<TeamMember>();
+    list.add(teamMember);
+    team.setCollaborators(list);
+    Boolean ret = teamService.add(team);
+    assertTrue(ret);
+
+    team.setTeamName("submarine_update");
+    TeamMember teamMember_update = new TeamMember();
+    teamMember_update.setTeamName("submarine");
+    teamMember_update.setInviter(0);
+    teamMember_update.setMember("zhulinhao");
+    teamMember_update.setCreateBy("createByteamMember2");
+    list.add(teamMember_update);
+
+    boolean editRet = teamService.updateByPrimaryKeySelective(team);
+    assertTrue(editRet);
+    List<Team> teamList = teamService.queryPageList("zhulinhao", "create_time", "desc", 0, 100);
+    assertEquals(teamList.size(), 1);
+
+    Team team_db = teamList.get(0);
+    assertEquals(team.getTeamName(), team_db.getTeamName());
+    assertEquals(team_db.getCollaborators().size(), 2);
+  }
+
+  @Test
+  public void delete() throws Exception {
+    TeamMember teamMember = new TeamMember();
+    teamMember.setTeamName("submarine");
+    teamMember.setInviter(0);
+    teamMember.setMember("admin");
+    teamMember.setCreateBy("createByteamMember");
+
+    Team team = new Team();
+    team.setTeamName("submarine");
+    team.setOwner("zhulinhao");
```

Comment:
Cannot appear the name of the contributor, Change to a common name, e.g., submarine1, submarine2 ...